sfs/time/drivingfunction.py

```diff
@@ -255,3 +256,194 @@ def apply_delays(signal, delays):
     for column, row in enumerate(delays_samples):
         out[row:row + len(data), column] = data
     return util.DelayedSignal(out, samplerate, offset_samples / samplerate)
+
+
+def nfchoa_25d_plane(x0, r0, npw, max_order=None, c=None, fs=44100, normalize=True):
+    """Vritual plane wave by 2.5-dimensional NFC-HOA.
+
+    Parameters
+    ----------
+    x0 : (N, 3) array_like
+        Sequence of secondary source positions.
+    r0 : float
+        Radius of the circular secondary source distribution
+    npw : (3,) array_like
+        Unit vector (propagation direction) of plane wave.
+    max_order : int
+        Ambisonics order
+    c : float, optional
+        Speed of sound
+
+    Returns
+    -------
+    delay : float
+        Overall delay (common to all secondary source)
+    weight : float
+        Overall weight (common to all secondary sources)
+    sos : list
+        Second-order section filters
+    phaseshift : float
+        Phase shift
+
+    References
+    ----------
+        S. Spors, V. Kuscher, J. Ahrens (2011) - "Efficient realization of
+       model-based rendering for 2.5-dimensional near-field compensated higher
+       order Ambisonics", WASPAA, p. 61-64
+
+       See Eq.(10)
```

Comment:
Please move the reference to the reference page.

Reply:
The extra reference pages was not part of this branch before. I merged it with the latest version of master and you can find the reference page at `doc/references.rst`.